docs/application/dotnet/guides/applications/uiapplication/ui-app.md

```diff
@@ -254,7 +221,6 @@ background category in its manifest file:
 </manifest>
 ```
 
-
-## Related Information
+## Related information
   * Dependencies
```

Comment:
Although the "*" appears as a bullet point in the staging area, requesting to use a dash "-" here for consistencies sake.
```suggestion
  - Dependencies
```